test/Splitter.t.sol

```diff
@@ -250,17 +263,22 @@ contract SplitterTest is DssTest {
         uint256 farmReward = vow.bump() * (WAD - splitter.burn()) / RAD;
         uint256 prevLastUpdateTime = farm.lastUpdateTime();
 
-        vm.expectEmit(false, false, false, false); // only check event signature (topic 0)
-        emit Kick(0, 0);
+        vm.expectEmit(false, false, false, true);
+        emit Kick(vow.bump(), vow.bump() * splitter.burn() / WAD, farmReward);
         vow.flap();
 
         assertEq(vat.dai(address(vow)), initialVowVatDai - vow.bump());
         assertEq(vat.dai(DAI_JOIN), initialDaiJoinVatDai + vow.bump());
         assertEq(vat.dai(address(splitter)), 0);
 
         assertEq(GemLike(DAI).balanceOf(UNIV2_DAI_MKR_PAIR), initialReserveDai + vow.bump() * splitter.burn() / RAD);
-        assertLt(GemLike(MKR).balanceOf(UNIV2_DAI_MKR_PAIR), initialReserveMkr);
-        assertGt(GemLike(MKR).balanceOf(address(PAUSE_PROXY)), initialMkr);
+        if (splitter.burn() == 0) {
+            assertEq(GemLike(MKR).balanceOf(UNIV2_DAI_MKR_PAIR), initialReserveMkr);
+            assertEq(GemLike(MKR).balanceOf(address(PAUSE_PROXY)), initialMkr);
+        } else {
+            assertLt(GemLike(MKR).balanceOf(UNIV2_DAI_MKR_PAIR), initialReserveMkr);
+            assertGt(GemLike(MKR).balanceOf(address(PAUSE_PROXY)), initialMkr);
+        }
```

Comment:
neat - would be nice to split in the same way in the burn and farm checks (currently first one uses if/else, the second uses ternary conditions).

Reply:
Rearranged in 79a0021c